.devcontainer/post-create.sh

```diff
@@ -1,7 +1,64 @@
 #!/bin/bash
+################################################################################
+### Script deploying the Observ-K8s environment
+### Parameters:
+### clustername : only required if not triggered in a codespace; define the name of the environemnt
+### githubtoken : only requried if not triggered in a codespace
+### deploymentmode: if empty then it means it has been trigered by a codespace, otherwise you can pass any string : gke, aws, kind
```

Comment:
It took me some time to understand the `deploymentmode` flag and I'm still not sure if I got it. It describes where the Kubernetes cluster is deployed right? If yes, what's the difference between `kind` & `git`? If I understand it correclty, it could probably be a boolean flag (e.g. `deployLocalCluster`)?